cmd/txt-suite/cmd.go

```diff
@@ -2,17 +2,50 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"sort"
 	"strconv"
 	"strings"
+
+	"github.com/9elements/txt-suite/pkg/test"
 )
 
-var testno = flag.String("t", "", "Select test number 1 - 50. e.g.: -t 1,2,3,4,...")
+var testno = flag.String("t", "", "Select test number 1 - 50. e.g.: -t=1,2,3,4,...")
+var keeprunning = flag.Bool("i", false, "Errors will not stop the testing. Tests will keep going.")
```

Comment:
What does -i ?
If there's a fatal error we have to stop/crash.
If there's no fatal error we always keep going.